packages/framework/src/Command/RedisCleanCacheCommand.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Shopsys\FrameworkBundle\Command;
+
+use Redis;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class RedisCleanCacheCommand extends Command
+{
+    /**
+     * @var string
+     */
+    protected static $defaultName = 'shopsys:redis:clean-cache';
+
+    /**
+     * @var \Redis[]
+     */
+    private $redisClients;
+
+    /**
+     * @param \Redis[] $redisClients
+     */
+    public function __construct(array $redisClients)
+    {
+        $this->redisClients = $redisClients;
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this
+            ->setDescription('Cleans up redis cache');
+    }
+
+    /**
+     * @param \Symfony\Component\Console\Input\InputInterface $input
+     * @param \Symfony\Component\Console\Output\OutputInterface $output
+     */
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        foreach ($this->redisClients as $redis) {
+            $prefix = $redis->getOption(Redis::OPT_PREFIX);
+            $keys = $redis->keys('*');
+            $plainKeys = $this->removePrefixes($keys, $prefix);
+            $redis->del($plainKeys);
```

Comment:
This implementation can take a long time. I found better solution for me. If you use some default client, you dont need to find all keys and remove prefixes to be able to delete them. 

This is the method from facade in my project. You can specific custom pattern (bestselling_products, twig, etc.) after prefix and Id for example if you want to remove only one section in twig cache.
```
public function clearCacheByPattern(string $pattern = '', string $id = null): void
{
    $pattern = $this->cachePrefix . $pattern . '*';
    if ($id !== null) {
        $pattern .= $id . '*';
    }
    $this->redisClient->eval('return redis.call(\'del\', unpack(redis.call(\'keys\', ARGV[1])))', [$pattern]);
}
```

Reply:
As I said above, I think that should not be part of this PR, so it would be great if you could create Issue or new PR.

Reply:
I used this `eval` solution, so there is no need to move it to the issue